tests/atlocal.in

```diff
@@ -81,28 +83,33 @@ fi
 
 function setup_env()
 {
-    if [ -d testing ]; then
-        RPMTEST=${PWD}/testing
-        HOME=${RPMTEST}
+    if [ -d tree ]; then
+        RPMTEST=${PWD}/tree
         RPM_CONFIGDIR=${RPMTEST}/@RPM_CONFIGDIR@
-        export RPMTEST HOME RPM_CONFIGDIR
+        export RPMTEST RPM_CONFIGDIR
     fi
 }
 
 function sandbox()
 {
-    (unset RPM_CONFIGDIR RPM_POPTEXEC_PATH
-     cd ${RPMTEST} && \
-     FAKECHROOT_BASE="${RPMTEST}" fakechroot "$@"
-    )
+    local cmd=$1
+    shift
+    case $cmd in
+        exec)
+            (unset RPM_CONFIGDIR RPM_POPTEXEC_PATH
+             cd ${RPMTEST} && \
```

Comment:
Here's another case on unquoted RPMTEST. For whatever reason, it seems to be semi-consistently RPMTEST that is being unquoted. Amusingly, it's quoted on the next line though :smile:

Reply:
Yup, same goes with the use of curly braces, actually... I know they're needed in some cases such as when you have adjacent text that could be interpreted as part of the variable name, but I mix and match both styles quite a lot here :smile: 

I'll fix the quotes though.

Reply:
> That said, lets not make this an "oh no, stop ship" situation. Missing quotes only affect marginal use-cases (in my books, using spaces in development directory names equal shooting your own foot just to be sure). So don't go chasing every unquoted variable now, it's not a show-stopper at all.

Oh yup, same thoughts here... i.e. if you're spaces in your filesnames, you're already doing it wrong :smile:

Reply:
Hmm, I actually forgot to fix up *this* occurrence, but as we discussed, let's not focus on that now :smile: